internal/pkg/docker/dockercompose/svc.go

```diff
@@ -0,0 +1,104 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package dockercompose
+
+import (
+	"fmt"
+	"github.com/aws/copilot-cli/internal/pkg/manifest"
+	"github.com/compose-spec/compose-go/types"
+	"time"
+)
+
+func convertBackendService(service *types.ServiceConfig, port uint16) (*manifest.BackendServiceConfig, IgnoredKeys, error) {
```

Comment:
should this be an exported function? i'm trying to understand how a future command would use this package...

Reply:
This will actually be called by another function that will be exported instead! This is an internal helper for those functions, since it doesn't handle actually parsing the Compose yaml.